cmd/registry/core/get.go

```diff
@@ -109,13 +109,13 @@ func GetArtifact(ctx context.Context,
 	handler ArtifactHandler) (*rpc.Artifact, error) {
 	request := &rpc.GetArtifactRequest{}
 	if segments[3] == "" {
-		request.Name = "projects/" + segments[1]
+		request.Name = "projects/" + segments[1] + "/locations/global"
 	} else if segments[5] == "" {
-		request.Name = "projects/" + segments[1] + "/apis/" + segments[3]
+		request.Name = "projects/" + segments[1] + "/locations/global/apis/" + segments[3]
 	} else if segments[7] == "" {
-		request.Name = "projects/" + segments[1] + "/apis/" + segments[3] + "/versions/" + segments[5]
+		request.Name = "projects/" + segments[1] + "/locations/global/apis/" + segments[3] + "/versions/" + segments[5]
 	} else {
-		request.Name = "projects/" + segments[1] + "/apis/" + segments[3] + "/versions/" + segments[5] + "/specs/" + segments[7]
+		request.Name = "projects/" + segments[1] + "/locations/global/apis/" + segments[3] + "/versions/" + segments[5] + "/specs/" + segments[7]
```

Comment:
A general observation unrelated to this change: go/go-style/best-practices#string-concat-fmt

Reply:
Thanks! My old-school aversion to excessive function calls is showing... happy to change to fit the style guide, though

Reply:
This involves several lines of change in this file and probably others, so I'd prefer to address this in a future PR

Reply:
Sounds good :)